build/helper/documentation_helper.py

```diff
@@ -386,6 +386,7 @@ def _fix_references(node, doc, cfg, make_link=False):
     if 'enum' in node:
         config['start_enum'] = node['enum']
 
+    # We may have slashes before the underscore (NISCOPE\_ATTR\_SOMETHING)
```

Comment:
Seems simpler to first replace `\_\ with `_`, then go through the one regular expression.

Reply:
But there may be other `\_` that we don't want to replace.

Reply:
I don't know, but I am not confident enough in the documentation to definitively say there aren't any others.